rasa/cli/arguments/train.py

```diff
@@ -151,7 +153,7 @@ def add_augmentation_param(
     parser.add_argument(
         "--augmentation",
         type=int,
-        default=50,
+        default=TrainingTrackerProvider.get_default_config()['augmentation_factor'],
```

Comment:
Oh this isn't actually as bad as I thought now that I see it written out - actually happier with this now. Still think it's a bit uncomfortable but it's not the worst

Reply:
🥳